scripts/globals/status.lua

```diff
@@ -2292,6 +2292,7 @@ tpz.mobMod =
     ALLI_HATE           = 68, -- Range around target to add alliance member to enmity list.
     NO_LINK             = 69, -- If set, mob cannot link until unset.
     NO_REST             = 70, -- Mob cannot regain hp (e.g. re-burrowing antlions during ENM).
+    SEVERE_CHANCE       = 71, -- % chance to use a severe spell like death or impact
```

Comment:
there were a few gaps in the list we could fill in, such as 
```lua

    SIGHT_RANGE         = 4,  -- sight range
    SOUND_RANGE         = 5,  -- sound range
    BUFF_CHANCE         = 6,  -- % chance to buff (combat only)
    GA_CHANCE           = 7,  -- % chance to use -ga spell
    HEAL_CHANCE         = 8,  -- % chance to use heal
    HP_HEAL_CHANCE      = 9,  -- can cast cures below this HP %
    SUBLINK             = 10, -- sub link group
    LINK_RADIUS         = 11, -- link radius
    DRAW_IN             = 12, -- 1 - player draw in, 2 - alliance draw in -- only add as a spawn mod!
    -- 13 Available for use
```
13 would put it closer to the existing chance mods

Reply:
As discussed I'll leave this at 71 for now. The obsolete mods should be discussed independently.